core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -86,8 +86,13 @@ private[spark] class EventLoggingListener(
    * Creates the log file in the configured log directory.
    */
   def start() {
-    if (!fileSystem.isDirectory(new Path(logBaseDir))) {
-      throw new IllegalArgumentException(s"Log directory $logBaseDir does not exist.")
+    try{
+        if( !fileSystem.getFileStatus( new Path( logBaseDir ) ).isDir() ){
```

Comment:
Sorry, I'm losing my mind, I changed it to that, then I saw his, and I thought he had changed it. Should be right now.